app/models/institution.rb

```diff
@@ -13,11 +13,14 @@ class Institution < ActiveRecord::Base
   has_many :patients, dependent: :destroy
   has_many :samples, dependent: :destroy
   has_many :test_results, dependent: :destroy
+  has_many :roles, dependent: :destroy
 
   validates_presence_of :name
   validates_presence_of :kind
   validates_inclusion_of :kind, in: KINDS
 
+  after_create :create_predefined_roles
```

Comment:
How do we know which roles were created from the institution? We would need to add something in the database for this. Ideas?